### PR TITLE
Add basic security hub support

### PR DESCRIPTION
**Pull request by ryan-lane**
This change adds very basic support for security hub. Currently, it just adds a SecurityHub node type, which is associated with an account, if security hub is enabled for an account.

### CODE REVIEW

File: cartography/intel/aws/securityhub.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import boto3
+import neo4j
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
```
Comment (achantavy) [inline cartography/intel/aws/securityhub.py:15]: Also decorate with `@aws_handle_regions` from `cartography.util`

File: cartography/intel/aws/securityhub.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+def get_hub(boto3_session: boto3.session.Session) -> Dict:
+    client = boto3_session.client('securityhub')
+    try:
+        return client.describe_hub()
+    except client.exceptions.ResourceNotFoundException:
```
Comment (achantavy) [inline cartography/intel/aws/securityhub.py:20]: Maybe we should log a warning here too? Up to you

File: cartography/intel/aws/securityhub.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+@timeit
+def load_hub(
+    neo4j_session: neo4j.Session,
+    data: Dict,
+    current_aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    ingest_hub = """
+    WITH {Hub} AS hub
```
Comment (achantavy) [inline cartography/intel/aws/securityhub.py:32]: Could we use `UNWIND` here?

**Review by achantavy — CHANGES_REQUESTED** [review]
Some minor comments but looks good; thanks for the PR!

File: cartography/intel/aws/securityhub.py (hunk 2 above)
Comment (ryan-lane) [reply cartography/intel/aws/securityhub.py:20]: I think it'll throw an exception if it's not enabled for the account, so I think this is what we want.

File: cartography/intel/aws/securityhub.py (hunk 3 above)
Comment (ryan-lane) [reply cartography/intel/aws/securityhub.py:32]: There's only a single one per account/region.

File: cartography/intel/aws/securityhub.py (hunk 1 above)
Comment (ryan-lane) [reply cartography/intel/aws/securityhub.py:15]: This isn't regional. It's per account.

**Review by achantavy — APPROVED** [review]

**Review by achantavy — APPROVED** [review]

**Pull request merged**